src/lib/logger.rs

```diff
@@ -139,8 +142,8 @@ pub(crate) fn init(options: &LoggerOptions) {
             let record_level_fmt = get_formatted_log_level(&record_level, color);
 
             out.finish(format_args!(
-                "[{}] {} - {}",
-                &name_fmt, &record_level_fmt, &message
+                "[{}] {} {} - {}",
+                &name_fmt, recursion_level::recursion_level(), &record_level_fmt, &message
```

Comment:
That's actually a bug since now the recursion level will be determined when writing a log-entry (which can be after the increase e.g. when displaying the version).

Reply:
fixed in the latest push.